net/uring/netenc/unknown.go

```diff
@@ -0,0 +1,14 @@
+// +build !mips,!mips64,!ppc64,!s390x,!amd64,!386,!arm,!arm64,!mipsle,!mips64le,!ppc64le,!riscv64,!wasm
+
+package netenc
+
+import "C"
+
+// Ntohs converts network into native/host order.
+func Ntohs(v uint16) uint16 { return uint16(C.ntohs(C.uint16_t(v))) }
```

Comment:
I anticipate that we're going to attempt to get rid of the cgo dependency sooner rather than later. Let's omit this file entirely. If Go adds a new platform, compilation will fail, and we'll know to add it.